Engine/new_orc_parser.c

```diff
@@ -44,27 +45,19 @@ int32_t             closedir(DIR*);
 #endif
 
 extern void csound_orcrestart(FILE*, void *);
-
 extern int csound_orcdebug;
-
 extern void print_csound_predata(void *);
```

Comment:
General note for a part 2 pass, we'd probably benefit from reducing the use of extern in our codebase and organizing things into headers.

Reply:
I have moved all Engine/*.c prototypes to headers now.